testing/podman/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Simon Ser <simon.ser@intel.com>
+# Maintainer: Simon Ser <simon.ser@intel.com>
+pkgname=podman
+pkgver=1.5.1
+pkgrel=1
+pkgdesc="A daemonless container engine for developing, managing, and running OCI Containers"
+url="https://podman.io/"
+arch="all"
+license="Apache-2.0"
+makedepends="go bash btrfs-progs-dev lvm2-dev gpgme-dev libgpg-error-dev libseccomp-dev ostree-dev"
+depends="btrfs-progs cni-plugins device-mapper-libs go-md2man gpgme iptables ip6tables libassuan libgpg-error libseccomp conmon ostree runc skopeo"
+source="$pkgname-$pkgver.tar.gz::https://github.com/containers/libpod/archive/v$pkgver.tar.gz"
+
+build() {
+	cd "$srcdir/libpod-$pkgver"
```

Comment:
use builddir="$srcdir/libpod-$pkgver" in the root of the APKBUILD instead of cd'ing into it manually.